py/server/deephaven/table.py

```diff
@@ -518,21 +519,98 @@ def iter_rows(self, cols: Optional[Union[str, Sequence[str]]] = None, *, chunk_s
 
         Args:
             cols (Optional[Union[str, Sequence[str]]]): The columns to read. If None, all columns are read.
-            chunk_size (Optional[int]): The number of rows to read at a time. Default is 1, meaning one row at a time.
-                When chunk_size is greater than 1, the generator will yield a dictionary of column names to numpy arrays.
-                When chunk_size is 1, the generator will yield a dictionary of column names to scalar values.
 
         Returns:
-            A generator that yields a dictionary of column names to numpy arrays or scalar values.
+            A generator that yields a dictionary of column names to scalar values.
         """
-        from deephaven._table_reader import _table_reader_chunks, _table_reader_rows # to prevent circular import
+        from deephaven._table_reader import _table_reader_rows # to prevent circular import
 
-        if chunk_size == 1:
-            return _table_reader_rows(self, cols)
-        else:
-            return _table_reader_chunks(self, cols=cols, row_set=self.j_table.getRowSet(), chunk_size=chunk_size,
+        return _table_reader_rows(self, cols)
+
+    def iter_tuple(self, cols: Optional[Union[str, Sequence[str]]] = None, name: str = 'Deephaven') -> Generator[Tuple[str, Any], None, None]:
```

Comment:
I think we still need to give user option to specify `chunk_size`.

Also, this return type doesn't make sense to me; we are saying it's a 2-element tuple return, whereas I expect a tuple that has size equal to the number of columns. I think the appropriate tuple type is `tuple[Any, ...]`. (Maybe you meant to have some sort of `NamedTuple` type?)